modules/apps/segments/segments-web/src/main/java/com/liferay/segments/web/internal/odata/ExpressionVisitorImpl.java

```diff
@@ -338,6 +350,58 @@ private String _getPropertyName(Object object) {
 		return String.valueOf(object);
 	}
 
+	private Object _getSub(Object left, Object right) {
+		Duration duration = Duration.parse(right.toString());
+
+		if (MethodExpression.Type.NOW.equals(left)) {
+			long days = duration.toDays();
+
+			if (days <= 1) {
+				return "last24Hours";
```

Comment:
You should add a constants and translation keys or something in order to support last24hours.